src/blockchain/transactions.ts

```diff
@@ -39,6 +42,11 @@ export function getTxHash(state: TxState): string | undefined {
   return undefined;
 }
 
+export enum TxRebroadcastStatus {
```

Comment:
Sometimes I see that you use enumeration and sometimes  union types. How do you decide what to use and when?

Reply:
In general enums are handled better, so we should prefer them.
Although I don't think it makes sense to through all code and replace them.

Reply:
I was just asking.  IMO enums are better as well :)